hub/api/tests/test_dataset.py

```diff
@@ -810,7 +810,7 @@ def test_dataset_copy_s3_local():
     )
     for i in range(100):
         ds["num", i] = 2 * i
-    ds2 = ds.copy("s3://snark-test/cp_copy_data_s3_1")
+    ds2 = ds.copy("s3://snark-test/cp_copy_data_s3_1_a")
```

Comment:
I cleared out the corrupted datasets so you should be able to use the old paths if you want

Reply:
Ah. Already changed the paths on master, let's leave it as is